src/rpc/server.cpp

```diff
@@ -328,13 +328,6 @@ bool RPCIsInWarmup(std::string *outStatus)
     return fRPCInWarmup;
 }
 
-bool IsDeprecatedRPCEnabled(const std::string& method)
```

Comment:
This should not actually be moved from server to util, since it's accessing global state of the RPC server. I'm assuming there were link errors trying to build without this change, but these would be fixed by calling Chain::rpcEnableDeprecated (see below).

Reply:
reverted